### ISSUE TITLE
Windows Terminal - add drop down to select ESP-IDF

### ISSUE THREAD

**Issue by georgik** [espressif/idf-installer#19]
Add item to Windows Terminal drop down list to launch ESP-IDF directly.
Sample JSON:
```
{
          "guid": "{2c4de342-38b7-51cf-b940-2309a097f519}",
          "hidden": false,
          "name": "ESP-IDF 4.4",
          "startingDirectory": "C:/Users/USER/Desktop/esp-idf/",
          "commandline": "C:/WINDOWS/System32/WindowsPowerShell/v1.0/powershell.exe  -ExecutionPolicy Bypass -NoExit -File C:/Users/USER/.espressif/Initialize-Idf.ps1"
        }
```
The configuration file has JSONC format, so it's not possible to use just JSON parser/formatter.
The file is located in `$env:LocalAppData/Packages/Microsoft.WindowsTerminal_8wekyb3d8bbwe/LocalState/settings.json`

**Comment by georgik** [comment]
Better solution to store new extension configuration seems to be "JSON Fragments" which are supported in 1.7+ version of Windows Terminal.
But, for some reason it does not work and no entry appears in the  drop down menu.

**Comment by georgik** [comment]
Proper location of JSON file is not obvious from documentation.
It should be:
```
C:\Users\USER\AppData\Local\Microsoft\Windows Terminal\Fragments\ESP-IDF v4.2\framgent.json
```

**Comment by georgik** [comment]
Solved in installer release 2.9 beta 05
